plasmapy/particles/__init__.py

```diff
@@ -5,6 +5,7 @@
 # __all__ will be auto populated below
 __all__ = []
```

Comment:
I"m a little perplexed at the moment, since the `autodoc` is still trying to generate documentation for `_elements`, `_isotopes`, `_parsing`, and `_special_particles`.  (see https://plasmapy--1440.org.readthedocs.build/en/1440/particles/index.html#variables-attributes)  I'm confused on two fronts: (1) those modules, except for `_special_particles`, aren't exposed to the `plasmapy.particles` namespace and (2) the `autodoc` is interpreting them as attributes instead of modules.  ... I'll need to do some investigation.

Reply:
I think this would be a working solution...  You'd need to replace the code block for auto-populating `__all__` with

```python
# auto populate __all__
for name, obj in list(globals().items()):
    if inspect.ismodule(obj):
        continue

    if not (name.startswith("__") or name.endswith("__")):
        __all__.append(name)
__all__.sort()

del name, obj
```

This will just ensure modules (dirs and files) are not added to `__all__`.

Reply:
Thanks, this looks helpful!  I'll try this out hopefully later today.